### PR TITLE
Parallel checkout

### PR DESCRIPTION
**Pull request by tiberiuichim**

### CODE REVIEW

**Comment by tiberiuichim** [comment]
How to test the branch:

- make a clone
- go to a volto project, run: `path/to/mrs-developer/src/command.js --config=jsconfig.json --output=addons --fetch-https`

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 2.0.1 (unreleased)
```
Comment (ebrehault) [inline CHANGELOG.md:1]: As it is a new feature, I guess it would deserve a minor version: `2.1.0`

**Review by ebrehault — APPROVED** [review]

**Review by sneridagh — APPROVED** [review]

**Comment by ebrehault** [comment]
@tiberiuichim it is published on npm

**Comment by tiberiuichim** [comment]
@ebrehault Thank you very much!

**Pull request merged**